cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorState.java

```diff
@@ -500,45 +501,57 @@ public String getPlaintext() {
                              STATE, _state,
                              interBrokerPartitionMovementStats.get(ExecutionTaskState.PENDING),
                              interBrokerPartitionMovementStats.get(ExecutionTaskState.IN_PROGRESS),
                              interBrokerPartitionMovementStats.get(ExecutionTaskState.ABORTING),
                              numFinishedMovements(INTER_BROKER_REPLICA_ACTION),
                              numTotalMovements(INTER_BROKER_REPLICA_ACTION),
-                             _executionTasksSummary.finishedInterBrokerDataMovementInMB(),
-                             numTotalInterBrokerDataToMove(), _maximumConcurrentInterBrokerPartitionMovementsPerBroker,
+                             finishedInterBrokerDataMovementInMB,
+                             numTotalInterBrokerDataToMove,
+                             finishedInterBrokerDataPercent,
+                             _maximumConcurrentInterBrokerPartitionMovementsPerBroker,
                              _isTriggeredByUserRequest ? TRIGGERED_USER_TASK_ID : TRIGGERED_SELF_HEALING_TASK_ID, _uuid,
                              TRIGGERED_TASK_REASON, _reason, recentlyDemotedBrokers, recentlyRemovedBrokers);
       case INTRA_BROKER_REPLICA_MOVEMENT_TASK_IN_PROGRESS:
         intraBrokerPartitionMovementStats = _executionTasksSummary.taskStat().get(INTRA_BROKER_REPLICA_ACTION);
-        return String.format("{%s: %s, pending/in-progress/aborting/finished/total intra-broker partition movement %d/%d/%d/%d/%d, completed/total"
-                             + " bytes(MB): %d/%d, maximum concurrent intra-broker partition movements per-broker: %d, %s: %s, %s: %s%s%s}",
+        long finishedIntraBrokerDataMovementInMB = _executionTasksSummary.finishedIntraBrokerDataMovementInMB();
+        long numTotalIntraBrokerDataToMove = numTotalIntraBrokerDataToMove();
+        double finishedIntraBrokerDataPercent = ((double) finishedIntraBrokerDataMovementInMB) / numTotalIntraBrokerDataToMove * 100;
+        return String.format("{%s: %s, pending(%d)/in-progress(%d)/aborting(%d)/finished(%d)/total(%d) intra-broker partition movements,"
+                             + "completed(%d)/total(%d) bytes in MBs: %.2f%%, maximum concurrent intra-broker partition movements per-broker:"
+                             + "%d, %s: %s, %s: %s%s%s}",
                              STATE, _state,
                              intraBrokerPartitionMovementStats.get(ExecutionTaskState.PENDING),
                              intraBrokerPartitionMovementStats.get(ExecutionTaskState.IN_PROGRESS),
                              intraBrokerPartitionMovementStats.get(ExecutionTaskState.ABORTING),
                              numFinishedMovements(INTRA_BROKER_REPLICA_ACTION),
                              numTotalMovements(INTRA_BROKER_REPLICA_ACTION),
                              _executionTasksSummary.finishedIntraBrokerDataMovementInMB(),
```

Comment:
Replace this line with finishedIntraBrokerDataMovementInMB.

Reply:
Good call.